service/src/main/java/com/zebrunner/reporting/service/integration/tool/AbstractIntegrationService.java

```diff
@@ -41,6 +49,20 @@ public boolean isEnabledAndConnected(Long integrationId) {
 
     @SuppressWarnings("unchecked")
     public T getAdapterByIntegrationId(Long integrationId) {
+        IntegrationAdapter adapter = (IntegrationAdapter) getNotNullAdapterByIntegrationId(integrationId);
+        boolean isAdapterEnabledAndConnected = isEnabledAndConnected(adapter);
```

Comment:
minor note - avoid prefixing boolean variables with **is**. For boolean it is pretty much the same as get for other type getters, meaning that it is typically used for method returning boolean rather than variables.